introduction.md

```diff
@@ -1,2 +1,5 @@
 Hola soy Geostarter 1
-favorite path is JavaScript
\ No newline at end of file
+favorite path is JavaScript
+* Git
+* JavaScript
+* Java
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?